tests/src/mocks/index.ts

```diff
@@ -9,6 +9,16 @@ import { Tier } from '../../../src/core/users/MongoDBTiersRepository';
 const randomDataGenerator = new Chance();
 
 export default function getMocks() {
+  const user = {
```

Comment:
If you can use functions to instantiate mocks instead of having to create a const each time, that would be cleaner, easier to use and faster to work with. Example [here](https://github.com/internxt/drive-server-wip/blob/8ed79deac574946f542fe9b9f0b904a864c3864c/test/fixtures.ts#L171) where you can instantiate a user, by default it has as real as possible fields (using chance). It has of course things to improve, but it's far more scalable than this way of creating mocks.